docker/test/stateless/run.sh

```diff
@@ -262,14 +265,17 @@ function run_tests()
 
 export -f run_tests
 
+
+# This should be enough to setup job and collect artifacts
+TIMEOUT=$((MAX_RUN_TIME - 300))
 if [ "$NUM_TRIES" -gt "1" ]; then
     # We don't run tests with Ordinary database in PRs, only in master.
     # So run new/changed tests with Ordinary at least once in flaky check.
-    timeout_with_logging "$MAX_RUN_TIME" bash -c 'NUM_TRIES=1; USE_DATABASE_ORDINARY=1; run_tests' \
+    timeout_with_logging "$TIMEOUT" bash -c 'NUM_TRIES=1; USE_DATABASE_ORDINARY=1; run_tests' \
       | sed 's/All tests have finished//' | sed 's/No tests were run//' ||:
 fi
 
-timeout_with_logging "$MAX_RUN_TIME" bash -c run_tests ||:
+timeout_with_logging "$TIMEOUT" bash -c run_tests ||:
```

Comment:
I don't see that this helps here - https://s3.amazonaws.com/clickhouse-test-reports/66311/076d2bb960a797c6fb25fcf0d3e390b3d695de46/stateless_tests__debug__[2_2].html

Can you please take a look why?

Reply:
Please rebase, timeout is decreased and works

Reply:
Where? I don't see any $TIMEOUT adjustments apart from this PR

Reply:
https://github.com/ClickHouse/ClickHouse/commit/4a56c601b2b4ec364f808b25d6e9d9adfd4d3ce2

Reply:
How about this one https://s3.amazonaws.com/clickhouse-test-reports/66439/669ce7cd40881a075fe473027e6668d13683c840/stateful_tests__tsan_.html?

Reply:
There is no change for stateful tests because they use `stateful/run.sh`, which has a larger timeout. I will fix that as well.